packages/babel-plugin-proposal-private-methods/test/fixtures/private-method-loose/accessors/output.js

```diff
@@ -0,0 +1,46 @@
+var Foo =
+/*#__PURE__*/
+function () {
+  "use strict";
+
+  function Foo() {
+    babelHelpers.classCallCheck(this, Foo);
+    Object.defineProperty(this, _privateField, {
+      writable: true,
+      value: "top secret string"
+    });
+    Object.defineProperty(this, _get_privateGetter, {
+      value: _get_privateGetter2
+    });
+    Object.defineProperty(this, _set_privateSetter, {
+      value: _set_privateSetter2
+    });
```

Comment:
This should be
```js
Object.defineProperty(this, _privateGetter, {
  get: _get_privateGetter2
});
```
so that we can rely on the native getter/setter implementation. In case of a getter/setter pair, it should just be
```js
Object.defineProperty(this, _foo, {
  get: _get_foo,
  set: _set_foo,
});
```

Reply:
Ah, good point, will change.